src/libraries/Microsoft.PowerFx.Core/FeatureFlags.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.PowerFx
+{
+    /// <summary>
+    /// Hosts can enable these flags to try out early features. 
+    /// Any flags will eventually default to true and then get removed once the feature is finalized. 
+    /// </summary>
+    public static class FeatureFlags
+    {
+        /// <summary>
+        /// Enable String Interpolation feature. 
+        /// Added 12/3/2021
+        /// </summary>
+        public static bool StringInterpolation = false;
```

Comment:
Should these flags be 1, 2, 4, 8, etc instead of just booleans?

Reply:
Boolean is easier to check. We don't need to optimize storage with an enum.